apply-labels/labelers.example.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Bagus nih kalo kita bisa support output dari script create user

Reply:
yang ini, aku buat ada 2: 
1. as a separate command - jadi user bisa run convert dulu ke labelers.json, trus isi documents[] array ya
2. included in apply_row_answers - given user has populated the documents field, they can run the command with an additional parameter `--users_csv` - nanti credentialsnya bakalan diisi ke client_id & client_secret aja